relay-kafka/src/config.rs

```diff
@@ -85,41 +85,53 @@ impl KafkaTopic {
 
 /// Configuration for topics.
 #[derive(Serialize, Deserialize, Debug)]
-#[serde(default)]
+#[serde(default, deny_unknown_fields)]
 pub struct TopicAssignments {
     /// Simple events topic name.
+    #[serde(alias = "ingest-events")]
     pub events: TopicAssignment,
     /// Events with attachments topic name.
+    #[serde(alias = "ingest-attachments")]
     pub attachments: TopicAssignment,
     /// Transaction events topic name.
+    #[serde(alias = "ingest-transactions")]
     pub transactions: TopicAssignment,
     /// Outcomes topic name.
+    #[serde(alias = "outcomes")]
     pub outcomes: TopicAssignment,
     /// Outcomes topic name for billing critical outcomes. Defaults to the assignment of `outcomes`.
+    #[serde(alias = "outcomes-billing")]
     pub outcomes_billing: Option<TopicAssignment>,
     /// Session health topic name.
+    #[serde(alias = "ingest-sessions")]
     pub sessions: TopicAssignment,
-    /// Default topic name for all aggregate metrics. Specialized topics for session-based and
-    /// generic metrics can be configured via `metrics_sessions` and `metrics_generic` each.
-    pub metrics: TopicAssignment,
-    /// Topic name for metrics extracted from sessions. Defaults to the assignment of `metrics`.
-    pub metrics_sessions: Option<TopicAssignment>,
+    /// Topic name for metrics extracted from sessions, aka release health.
+    #[serde(alias = "metrics")]
```

Comment:
This is tricky, if both `metrics` and `ingest-metrics` are defined the precedence isn't clear (it was before). That being sad, I do like the change.

Reply:
This is intended to be very temporary just while we cut things over in production. After the migration I think we should be very strict and only support the actual topic names

Reply:
I meant, that if someone currently has both topics configured the precedence has now changed, which is a breaking change and this might affect us as well.

Reply:
I double checked ST and prod -- we don't use both in any of the environments we control at least.